actor/src/main/scala/org/apache/pekko/dispatch/ThreadPoolBuilder.scala

```diff
@@ -176,6 +176,9 @@ object MonitorableThreadFactory {
   private[pekko] class PekkoForkJoinWorkerThread(_pool: ForkJoinPool)
       extends ForkJoinWorkerThread(_pool)
       with BlockContext {
+
+    override def onTermination(exception: Throwable): Unit = super.onTermination(exception)
```

Comment:
Why would we need to override a function just to call its super class implementation?

Reply:
That's a hook where we can do aop around it, override it in subclass can limit the scope.

Reply:
I mean I can use `Byte-Buddy` to aop it then,

Reply:
I don't understand why this is not inherited anyway from the superclass.

Reply:
just tried, it works too.